core/modules/wbase/Task.h

```diff
@@ -63,10 +85,10 @@ struct Task {
 public:
     static std::string const defaultUser;
 
-    typedef std::shared_ptr<Task> Ptr;
-    typedef proto::TaskMsg_Fragment Fragment;
-    typedef std::shared_ptr<Fragment> FragmentPtr;
-    typedef std::shared_ptr<proto::TaskMsg> TaskMsgPtr;
+    using Ptr =  std::shared_ptr<Task>;
+    using Fragment = proto::TaskMsg_Fragment;
+    using FragmentPtr = std::shared_ptr<Fragment>;
+    using TaskMsgPtr = std::shared_ptr<proto::TaskMsg>;
 
     struct ChunkEqual {
```

Comment:
Hmm, how are these functors used?  Should they really be part of the public Task interface?   Are they good candidates to move to e.g. anon. namespace of some implementation file?